src/main.py

```diff
@@ -1,9 +1,23 @@
+"""
+Script to determine leader and follower roles.
+
+Takes input from the camera to determine if the current vehicle
+should lead or follow based on whether or not there exists an ARTag
+in front of the vehicle.
+
+If an ARTag exists, the vehicle will become a follower vehicle.
+If an ARTag does not exist, the vehicle will become the leader vehicle.
+
+Authors: Zein Youssef and Steven Dropiewski
+"""
+
```

Comment:
should there be a extra line here below the author? Its not consistent with the 2 files above.

Reply:
I think I was _going_ to do this for all of them and decided I wouldn't. Fixing now.